src/components/FilterList.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { firestore } from '../lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+let arrString = ['fruit', 'vegetables', 'water', 'legumes'];
```

Comment:
What is this used for..? It seems we're filtering out of this array, but shouldn't we be referencing the list of items from firebase?